probe/bundle.go

```diff
@@ -91,6 +91,22 @@ func (p *Bundle) ActiveProbes() []string {
 	return activeProbes
 }
 
+// GetStatus returns the status of all the probes
+func (p *Bundle) GetStatus() map[string]interface{} {
+	p.RLock()
+	defer p.RUnlock()
+
+	status := make(map[string]interface{})
+	for k, v := range p.Handlers {
+		if v, ok := v.(StatusReporter); ok {
+			status[k] = v.GetStatus()
```

Comment:
maybe it should be part of the `Handler` interface, so that each probe handler will have to implement it